src/app/tests/suites/certification/Test_TC_MC_1_10.yaml

```diff
@@ -43,49 +43,65 @@ tests:
           constraints:
               type: uint16
 
+    #Issue 10988
     - label: "Read the global attribute: AttributeList"
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AttributeList"
       response:
+          value: [0, 1, 2, 3, 4, 5, 6, 7, 65528, 65529, 65531, 65533]
           constraints:
               type: list
 
     #issue #11053 disabled steps below Global attributes missing from YAML framework
-    - label: "Read the global attribute: EventList"
-      disabled: true
-      command: "readAttribute"
-      attribute: "EventList"
-      response:
-          constraints:
-              type: list
+    - label:
+          "Read EventList attribute from the DUT and Verify that the DUT
+          response provides a list of supported events."
+      verification: |
+          Not implemented in chip-tool
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
 
+    #Issue 10988
     - label: "Read the global attribute: AcceptedCommandList"
-      disabled: true
+      PICS: PICS_SKIP_SAMPLE_APP
       command: "readAttribute"
       attribute: "AcceptedCommandList"
       response:
+          value: [0, 1]
           constraints:
               type: list
 
+    #Issue 17459
```

Comment:
The issue and test plan don't match the spec.

Reply:
Opened testplan issue https://github.com/CHIP-Specifications/chip-test-plans/issues/1107